src/index.ts

```diff
@@ -10,10 +10,8 @@ import {OutgoingHttpHeaders} from 'http';
 const jsonBigint = require('json-bigint'); // eslint-disable-line
 
 export const HOST_ADDRESS = 'http://169.254.169.254';
-export const BASE_PATH = '/computeMetadata/v1';
-export const BASE_URL = HOST_ADDRESS + BASE_PATH;
 export const SECONDARY_HOST_ADDRESS = 'http://metadata.google.internal.';
```

Comment:
Actually, I just noticed this. why does the URL have a trailing dot?

Reply:
@bojeil-google I believe the reason that the `.` exists has to do with DNS caching, it makes the lookup significantly faster in some GCP environments to have the trailing dot.

tldr; [fully qualified domain names do have a `.`](http://www.dns-sd.org/TrailingDotsInDomainNames.html), but it's usually treated as optional (by most libraries, browsers, etc)

Reply:
Ah, makes sense. Thanks for clarifying that.